### ISSUE TITLE
The colour of the buttons doesn't fit the theme of the site

### ISSUE THREAD

**Issue by adrianbanks** [dddeastanglia/DDDEastAnglia#85]
They are currently blue. These styles work much better (and match the banner colours):

```
.btn-primary {
    background-color: #008600;
    background-image: -moz-linear-gradient(top,#00a400,#005800);
    background-image: -webkit-gradient(linear,0 0,0 100%,from(#00a400),to(#005800));
    background-image: -webkit-linear-gradient(top,#00a400,#005800);
    background-image: -o-linear-gradient(top,#00a400,#005800);
    background-image: linear-gradient(to bottom,#00a400,#005800);
    background-repeat: repeat-x;
    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ff00a400',endColorstr='#ff005800',GradientType=0);
    border: 1px solid #001b00;
    -webkit-border-radius: 4px;
    -moz-border-radius: 4px;
    border-radius: 4px;
    -webkit-box-shadow: 0 1px 4px rgba(0,0,0,.065);
    -moz-box-shadow: 0 1px 4px rgba(0,0,0,.065);
   box-shadow: 0 1px 4px rgba(0,0,0,.065);    
}

.btn-primary:hover {
    background-color: #006600;
}
```

but I'm not sure how to update the base bootstrap stylesheet to include them. @alastairs Any ideas?

**Comment by alastairs** [comment]
You need to regenerate the stylesheet [here](http://twitter.github.io/bootstrap/customize.html) and be sure to include [the full list of customisations](https://github.com/dddeastanglia/DDDEastAnglia/issues/37#issuecomment-16233986).

**Comment by alastairs** [comment]
I'm also wary of using a green colour for the primary action button, because "success" buttons are also green.

**Comment by adrianbanks** [comment]
> "success" buttons are also green

Is that just a bootstrap convention?

I see your point though. Any suggestions for a better colour?

**Comment by alastairs** [comment]
> Is that just a bootstrap convention?

Yes. Anything "success" (alerts, buttons, etc.) are styled in green, mostly the same shade, or a related one. 

> Any suggestions for a better colour?

I think blue is quite a good colour for this sort of thing because it stands out from the rest of the site, inviting you to click the button. I take your point that the current shade doesn't fit, though, so perhaps it's just a case of finding a lighter shade? Be careful to avoid the Info button colour too, which is a sort of aquamarine blue. 

http://twitter.github.io/bootstrap/base-css.html#buttons

**Comment by adrianbanks** [comment]
Leaving this for now. We'll look at it for next time.
